pkg/executor/alisa.go

```diff
@@ -60,7 +61,26 @@ func (s *alisaExecutor) submitAlisaTask(submitCode, codeResourceURL, paramsResou
 		return fmt.Errorf("PAI task failed, please go to check details error logs in the LogViewer website: %s", strings.Join(pickPAILogViewerURL(b.String()), "\n"))
 	}
 	return nil
+}
+
+func (s *alisaExecutor) submitAlisaPyODPSTask(submitCode, args string) error {
```

Comment:
I would suggest merging `submitAlisaTask` to reduce code.

Reply:
Good idea, updated.